backend/ibutsu_server/constants.py

```diff
@@ -229,6 +229,49 @@
         ],
         "type": "widget",
     },
+    "importance-component": {
+        "id": "importance-component",
+        "title": "Importance by component",
+        "description": "Test results filtered by component and broken down by importance",
+        "params": [
+            {
+                "name": "job_name",
+                "description": "The name of the jenkins job to pull from",
+                "type": "string",
+                "required": True,
+                "default": "",
+            },
+            {
+                "name": "group_field",
+                "description": "the field in a result to group by, typically 'component'",
+                "type": "string",
+                "required": True,
+                "default": "component",
+            },
+            {
+                "name": "env",
+                "description": "The environment to filter by",
+                "type": "string",
+                "required": False,
+                "default": "",
+            },
+            {
+                "name": "components",
+                "description": "The component(s) to filter by",
+                "type": "string",
+                "required": False,
```

Comment:
Can we require this field? Because if it's empty, widget will not show info and it might be a bit confusing as widget requires at least 1 component to show data